### DISCUSSION TITLE
Treetable:  Transition Options Possible?

### DISCUSSION THREAD

**Discussion by tbaro** [primefaces/community#306] [PrimeNG] [answered]
Is there some way to cause a treetable with scrollable/frozen columns to "ease in" the additional rows when an onexpand event is triggered, and to "ease out"when an oncollapse event is triggered?  By "ease in" I mean slowly expand the row from a small height to a larger height over a specified transition period. (Vice versa on collapse)

An example tree table with variable height rows is attached. It would be great if someone could show how to apply such a feature to this sample code.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, treetable component doesn't offer this as a feature and it's not possible to workaround it since the animation doesn't exist in these functionalities. Also, please share Stackblitz example instead of unanimous zip files.
